### PR TITLE
Use named export to export each key code separately

### PR DESCRIPTION
**Pull request by mesaugat**
Fixes #10

### CODE REVIEW

File: src/KeyCode.js [hunk 1]
```diff
@@ -1,128 +1,119 @@
-export default (() => {
+import * as KeyCode from './KeyCode';
```
Comment (mesaugat) [inline src/KeyCode.js:1]: This is kind of hackish but works fine. Exporting itself for a default export at the end.

Comment (kabirbaidhya) [reply src/KeyCode.js:1]: Haven't seen this self-import thing yet. LOL :smile:. Let's wait to see if all tests pass.

Comment (mesaugat) [reply src/KeyCode.js:1]: All tests pass. 💚

**Review by kabirbaidhya — APPROVED** [review]
Looks good :+1:

**Pull request merged**